proto/eth/service/validator_service.proto

```diff
@@ -222,19 +222,6 @@ service BeaconValidator {
     };
   }
 
-  // ProduceSyncCommitteeContribution requests that the beacon node produces a sync committee contribution.
-  //
-  // HTTP response usage:
-  //  - 200: Successful response
-  //  - 400: Invalid request syntax
-  //  - 500: Beacon node internal error
-  //  - 503: Beacon node is currently syncing, try again later
-  //
-  // Spec: https://ethereum.github.io/beacon-APIs/?urls.primaryName=v2.3.0#/Validator/produceSyncCommitteeContribution
-  rpc ProduceSyncCommitteeContribution(v2.ProduceSyncCommitteeContributionRequest) returns (v2.ProduceSyncCommitteeContributionResponse) {
```

Comment:
Just to double check, are the request and response types still used anywhere? If no, we can also remove their proto definitions.

Reply:
Done